.github/ISSUE_TEMPLATE/config.yml.handlebars

```diff
@@ -0,0 +1,3 @@
+This file is automatically added by @npmcli/template-oss. Do not edit.
+
+blank_issues_enabled: true
```

Comment:
It looks like this file got created accidentally.

Reply:
good catch! oops!